editor/src/components/navigator/navigator-item/navigator-item.tsx

```diff
@@ -183,7 +183,7 @@ const computeResultingStyle = (
   return result
 }
 
-function isFocused(focusedElementPath: ScenePath | null, path: TemplatePath): boolean {
+export function isFocused(focusedElementPath: ScenePath | null, path: TemplatePath): boolean {
```

Comment:
Rather than exporting this function here, it would make more sense to move the function to a more core shared file (`template-path.ts` would work). Otherwise we are creating a direct dependency between the Canvas Controls code and the Navigator code (which isn't something we should do)